### PR TITLE
rotational_cipher

### PR DESCRIPTION
**Pull request by amymhaddad**

### CODE REVIEW

File: python/rotational-cipher/rotational_cipher.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    list_letters = list(LETTERS)
+    list_letters.append(' ')
+
+    for letter in text:
+        if letter in LIST_PUNCTUATION or letter in LIST_NUMBERS or letter == ' ':
+            new_text += letter
+        else:
+            index = list_letters.index(letter.lower())
+            rotational_cipher_formula = (index + key) % 26
```
Comment (paulghaddad) [inline python/rotational-cipher/rotational_cipher.py]: I'd suggest naming this more accurately: `index_after_rotation`, or something like that.

Comment (paulghaddad) [reply python/rotational-cipher/rotational_cipher.py]: Nice job on this formula!

File: python/rotational-cipher/rotational_cipher.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            new_letter = list_letters[rotational_cipher_formula]
+            if letter.isupper():
```
Comment (paulghaddad) [inline python/rotational-cipher/rotational_cipher.py]: Excellent!

**Comment by amymhaddad** [comment]
@paulghaddad Updates are made and ready for final review!

File: python/rotational-cipher/rotational_cipher.py (hunk 1 above)
Comment (paulghaddad) [reply python/rotational-cipher/rotational_cipher.py]: I'm sorry, I meant `(index + key) % 26` should be named `index_after_rotation`.

File: python/rotational-cipher/rotational_cipher.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+"""Provide the encryption result for a given text"""
+
+import string
```
Comment (paulghaddad) [inline python/rotational-cipher/rotational_cipher.py]: I thought there should be a line break after your imports, but I guess if Pylint didn't complain, it's fine.

File: python/rotational-cipher/rotational_cipher.py (hunk 1 above)
Comment (amymhaddad) [reply python/rotational-cipher/rotational_cipher.py]: @paulghaddad I think your variable name suggestion is a bit confusing. In my mind `rotational_cipher_formula` is just that: the formula to determine the rotation. I understand that it's being used as an index in the next line (line 22), but I think it's current name is most representative of what it is. 

Also, I restored the variable name in line 22 back to its original name.

**Pull request merged**